### PR TITLE
validator(dm): fix unstable test

### LINKED ISSUE #6734: test(dm): unstable test validator_basic failover
https://ci2.pingcap.net/blue/organizations/jenkins/dm_ghpr_integration_test/detail/dm_ghpr_integration_test/8862/pipeline

We tried to trigger two flush checkpoints, but only one occurred.
image

possible root cause:
8jxMpgq1dH

### PR DESCRIPTION
**Pull request by buchuitoudegou**
### What problem does this PR solve?

Issue Number: close #6734 

### What is changed and how it works?
1. wait for min-flush-interval to ensure the second flush happens
2. ensure the checkpoint is successfully flushed

### Release note 

```release-note
`None`
```

### CODE REVIEW

File: dm/tests/validator_basic/run.sh [hunk 1]
```diff
@@ -546,10 +546,14 @@ function stopped_validator_fail_over() {
 		"new\/ignored\/resolved: 1\/0\/0" 1 \
 		"\"stage\": \"Running\"" 2
 	# make sure validator checkpoint is flushed
+	sleep 1 # wait for the min flush interval
 	trigger_checkpoint_flush
 	run_sql_tidb_with_retry "select concat_ws('/', procd_ins, procd_upd, procd_del) processed
 														from dm_meta.test_validator_checkpoint where source='mysql-replica-01'" \
 		"processed: 3/2/1"
+	run_sql_tidb_with_retry "select count(1) cnt
```
Comment (lance6716) [inline dm/tests/validator_basic/run.sh:554]: just noticed why there's so many spaces 😂

Comment (buchuitoudegou) [reply dm/tests/validator_basic/run.sh:554]: copy from https://github.com/pingcap/tiflow/pull/6735.

Seems to try formatting the SQL better.

Comment (lance6716) [reply dm/tests/validator_basic/run.sh:554]: OK, in IDE it has good indent, but on GitHub webpage it doesn't

**Review by lance6716 — APPROVED** [review]

**Comment by lance6716** [comment]
PTAL @D3Hunter @lichunzhu

**Review by lichunzhu — APPROVED** [review]

**Pull request merged**